src/frontend/admin/src/components/templates/products/form/translations.ts

```diff
@@ -166,6 +166,17 @@ export const productFormMessages = defineMessages({
     defaultMessage: "Add target courses",
     description: "Title for the add target courses modal",
   },
+  choiceTargetCourseCourseRunModalTitle: {
+    id: "components.templates.products.form.translations.choiceTargetCourseCourseRunModalTitle",
+    defaultMessage: "Selection of course runs",
+    description: "Title for the course runs selection section",
+  },
+  choiceTargetCourseCourseRunModalAlertContent: {
+    id: "components.templates.products.form.translations.choiceTargetCourseCourseRunModalAlertContent",
+    defaultMessage:
+      "You can now select course runs by activating this switch. If you leave this switch disabled, all course runs will be selected by default.",
```

Comment:
WYT? 

`By default all course runs are selected, turn this switch on if you want to choose which course runs are selected.`